shared/templates/kernel_module_disabled/ansible.template

```diff
@@ -9,7 +9,7 @@
     dest: "/etc/modprobe.d/{{{ KERNMODULE }}}.conf"
     regexp: 'install\s+{{{ KERNMODULE }}}'
     line: "install {{{ KERNMODULE }}} /bin/true"
-{{% if product in ["sle12", "sle15"] or 'ol' in product or 'rhel' in product %}}
+{{% if product in ["sle12", "sle15"] or 'ol' in product or 'rhel' in product or 'ubuntu' in product %}}
```

Comment:
nice catch, can you also add this in the bash.template, it is currently missing

Reply:
thanks